warframe/worldstate/models/event.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from typing import List, Optional
+
+from msgspec import field
+
+from .reward import Reward
+
+from ..common import MultiQueryModel
+from ..enums import MissionType, Faction, Syndicate
+
+__all__ = ["Event"]
+
+
+class Event(MultiQueryModel):
+    activation: Optional[datetime] = None
+    "The time the Event started"
```

Comment:
it's your code, but i don't love the members being completely unspaced with the docstrings...